src/command-registry.coffee

```diff
@@ -18,6 +18,11 @@ SequenceCount = 0
 # command event listeners globally on `atom.commands` and constrain them to
 # specific kinds of elements with CSS selectors.
 #
+# Command names must follow the `namespace:event` pattern, where both parts can
+# be further split into words by hyphens. Typically `namespace` will be the
+# name of your package, and `event` describes the behavior of your command.
+# E.g. `awesome-package:toggle`.
```

Comment:
I'd also select a multi-word command name to drive the point home that both components should be dasherized.